web/impact/impact/v1/views/reserve_office_hour_view.py

```diff
@@ -210,3 +220,33 @@ def _response(self):
             'header': self.header,
             'detail': self.detail,
             'timecard_info': self.timecard_info})
+
+    def get_calendar_data(self, counterpart_name):
+        title = self.OFFICE_HOUR_TITLE.format(counterpart_name)
+        office_hour = self.office_hour
+        if office_hour.location is None:
+            timezone = "UTC"
+            location = "MassChallenge"
+        else:
+            timezone = office_hour.location.timezone
+            location = office_hour.location
+        meeting_info = office_hour.meeting_info
+        location_info = "{location};{meeting_info}"
+        location_info = location_info.format(location=location,
+                                             meeting_info=meeting_info)
+        return Add2Cal(
+            start=office_hour.start_date_time.strftime(
+                ADD2CAL_DATE_FORMAT),
+            end=office_hour.end_date_time.strftime(
+                ADD2CAL_DATE_FORMAT),
+            title=title,
+            description=self._get_description(),
+            location=location_info,
+            timezone=timezone).as_dict()
```

Comment:
TypeError: unexpected keyword argument `timezone`

Reply:
Should be resolved now